MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.FramebufferHelper.cs

```diff
@@ -34,48 +34,88 @@ namespace Microsoft.Xna.Framework.Graphics
+                {
+                    _instance = new FramebufferHelperEXT(gd);
+                }
+#endif
+                else
+                {
+                    throw new PlatformNotSupportedException(
+                        "MonoGame requires either ARB_framebuffer_object or EXT_framebuffer_object." +
+                        "Try updating your graphics drivers.");
+                }
+
+                return _instance;
+            }
+
+            public static FramebufferHelper Get()
+            {
+                if (_instance == null)
+                    throw new InvalidOperationException("The FramebufferHelper has not been created yet!");
+                return _instance;
+            }
+
+            #endregion
+
+#if GLES
             public bool SupportsInvalidateFramebuffer { get; private set; }
 
             public bool SupportsBlitFramebuffer { get; private set; }
 #if IOS
 			internal const string OpenGLLibrary = ObjCRuntime.Constants.OpenGLESLibrary;
 #elif ANDROID
             internal const string OpenGLLibrary = "libGLESv2.dll";
+
             [DllImport("libEGL.dll", EntryPoint = "eglGetProcAddress")]
             public static extern IntPtr EGLGetProcAddress(string funcname);
 #endif
+
             #region GL_EXT_discard_framebuffer
 
-            internal const All AllColorExt = (All)0x1800;
-            internal const All AllDepthExt = (All)0x1801;
-            internal const All AllStencilExt = (All)0x1802;
+            internal const All AllColorExt = (All) 0x1800;
+            internal const All AllDepthExt = (All) 0x1801;
+            internal const All AllStencilExt = (All) 0x1802;
 
             [SuppressUnmanagedCodeSecurity]
             [DllImport(OpenGLLibrary, EntryPoint = "glDiscardFramebufferEXT", ExactSpelling = true)]
-            internal extern static void GLDiscardFramebufferExt(All target, int numAttachments, [MarshalAs(UnmanagedType.LPArray)] All[] attachments);
+            internal extern static void GLDiscardFramebufferExt(All target, int numAttachments,
+                [MarshalAs(UnmanagedType.LPArray)] All[] attachments);
 
             #endregion
 
             #region GL_APPLE_framebuffer_multisample
 
-            internal const All AllFramebufferIncompleteMultisampleApple = (All)0x8D56;
-            internal const All AllMaxSamplesApple = (All)0x8D57;
-            internal const All AllReadFramebufferApple = (All)0x8CA8;
-            internal const All AllDrawFramebufferApple = (All)0x8CA9;
-            internal const All AllRenderBufferSamplesApple = (All)0x8CAB;
+            internal const All AllFramebufferIncompleteMultisampleApple = (All) 0x8D56;
+            internal const All AllMaxSamplesApple = (All) 0x8D57;
+            internal const All AllReadFramebufferApple = (All) 0x8CA8;
+            internal const All AllDrawFramebufferApple = (All) 0x8CA9;
+            internal const All AllRenderBufferSamplesApple = (All) 0x8CAB;
```

Comment:
Can we revert these whitespace changes here?  It makes is hard to actually see the diffs.